personal_profile/templates/personal_profile/profile_template.html

```diff
@@ -0,0 +1,78 @@
+{% load static %}
```

Comment:
We may remove this file, since we use `feed/templates/feed/base_template.html`.

Reply:
See #33 about this.

Reply:
@taljacob2 I've seen and updated accordingly, great changes!